docs/index.md

```diff
@@ -26,7 +26,7 @@ The WordPress Operator is a member of the Ubuntu family. It's an open-source pro
 - [Code of conduct](https://ubuntu.com/community/code-of-conduct)
 - [Get support](https://discourse.charmhub.io/)
 - [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
-- [Contribute](Contribute)
+- [Contribute](https://github.com/canonical/wordpress-k8s-operator/blob/a5d7e240b47c0d2cfe9c12f88adbf6ab93a48b80/CONTRIBUTING.md)
```

Comment:
Out of curiosity: Why not linking to main?
https://github.com/canonical/wordpress-k8s-operator/blob/main/CONTRIBUTING.md

Reply:
I grabbed the permalink (for the main branch) instead -- do you think it should be changed?

Reply:
Your link is pointing to a specific commit, what Amanda proposes is to link to the last commit of the main branch (so that people clicking on this get the latest version)

Reply:
Ah ok, thanks for explaining. I'll update it then